public/controllers/management/management.js

```diff
@@ -207,26 +205,40 @@ export class ManagementController {
    * When controller loads
    */
   $onInit() {
-    this.clusterInfo = AppState.getClusterInfo();
+    try {
+      this.clusterInfo = AppState.getClusterInfo();
 
-    if (this.shareAgent.getAgent() && this.shareAgent.getSelectedGroup()) {
-      this.tab = 'groups';
-      this.switchTab(this.tab);
-      return;
-    }
+      if (this.shareAgent.getAgent() && this.shareAgent.getSelectedGroup()) {
+        this.tab = 'groups';
+        this.switchTab(this.tab);
+        return;
+      }
 
-    const location = this.$location.search();
+      const location = this.$location.search();
 
-    if (location && location.tab) {
-      this.tab = location.tab;
-      this.switchTab(this.tab);
-    }
+      if (location && location.tab) {
+        this.tab = location.tab;
+        this.switchTab(this.tab);
+      }
 
-    this.uploadFilesProps = {
-      msg: this.$scope.mctrl.rulesetTab,
-      path: `etc/${this.$scope.mctrl.rulesetTab}`,
-      upload: (files) => this.uploadFiles(files, this.$scope.mctrl.rulesetTab)
-    };
+      this.uploadFilesProps = {
+        msg: this.$scope.mctrl.rulesetTab,
+        path: `etc/${this.$scope.mctrl.rulesetTab}`,
+        upload: (files) => this.uploadFiles(files, this.$scope.mctrl.rulesetTab),
+      };
+    } catch (error) {
+      const errorOptions = {
+        level: UI_LOGGER_LEVELS.ERROR,
+        severity: UI_ERROR_SEVERITIES.BUSINESS,
+        error: {
+          error: error,
+          message: error?.message || '',
+          title: 'Error restarting cluster',
+        },
+      };
+
+      ErrorOrchestratorService().handleError(errorOptions);
```

Comment:
Check the ultimates changes of `4.3-7.10` now you can use this service like this:
`getErrorOrchestrator().handleError(errorOptions);`